great_expectations/expectations/expectation.py

```diff
@@ -951,11 +949,11 @@ def build_configuration(cls, *args, **kwargs):
     def run_diagnostics(
         self,
         raise_exceptions_for_backends: bool = False,
-        force_no_progress_bar: bool = True,
         ignore_suppress: bool = False,
         ignore_only_for: bool = False,
         debug_logger: Optional[logging.Logger] = None,
         only_consider_these_backends: Optional[List[str]] = None,
+        context: Optional[Any] = None,  # Cannot type DataContext due to circular import
```

Comment:
@kenwade4 You can type it in quotes and then also add `# noqa: F812` -- here is an example:
```
Optional["Validator"]:  # noqa: F821
```
Also, please feel free to use the proper exact `DataContext` class (is it really `DataContext`?  or is it `AbstractDataContext`, etc.?) 

Thanks!

Reply:
Yea it's a full-on DataContext object returned from `DataContext.create(...)`